inst/tests/tests.Rraw

```diff
@@ -18330,3 +18330,22 @@ if (test_bit64) {
   apple = data.table(id = c("a", "b", "b"), time = c(1L, 1L, 2L), y = i64v[1:3])
   test(2248, dcast(apple, id ~ time, value.var = "y"), data.table(id = c('a', 'b'), `1` = i64v[1:2], `2` = i64v[4:3], key='id'))
 }
+
+# fread(...,fill) can also be used to specify a guess on the maximum number of columns #2691 #1812 #4130 #3436 #2727
+dt_str = paste(rep(c("1,2\n", "1,2,3\n"), each=100), collapse="")
```

Comment:
I'd use `strrep()`, either:

```suggestion
dt_str = paste0(strrep("1,2\n", 100L), strrep("1,2,3\n", 100L))
dt_str = paste(rep(c("1,2\n", "1,2,3\n"), each=100), collapse="")
```

or

```suggestion
dt_str = paste(strrep(c("1,2\n", "1,2,3\n"), 100L), collapse="")
dt_str = paste(rep(c("1,2\n", "1,2,3\n"), each=100), collapse="")
```

(looks like comments don't accept multiple ```` ```suggest ```` blocks...)

Reply:
`strrep` is introduced in R 3.3.0 feature

Reply:
Bummer. Let's add a TODO to use it. We can also use a local equivalent for now to make the switch easier later, e.g.

```r
strrep = function(x, times) mapply(function(x, times) paste(rep(x, times), collapse=""), rep_len(x, l <- pmax(length(x), length(times))), rep_len(times, l), USE.NAMES=FALSE)
```

(not strictly necessary)

Reply:
NVM, it's overkill.